Source/cryptography/implementation/vault_implementation.h

```diff
@@ -27,14 +27,14 @@
 extern "C" {
 #endif
 
-struct VaultImplementation;
 
+struct VaultImplementation;
 
 struct VaultImplementation* vault_instance(const enum cryptographyvault id);
 
 uint16_t vault_size(const struct VaultImplementation* vault, const uint32_t id);
 
-uint32_t vault_import(struct VaultImplementation* vault, const uint16_t length, const uint8_t blob[]);
+uint32_t vault_import(struct VaultImplementation* vault, const uint16_t length, const uint8_t blob[],const bool blobIsName);
```

Comment:
Is the isBlobName parameter still neccessary? I think it breaks compilation now.

Reply:
Yes sorry , missed that out from the prev implementation , will make that change and other changes as well.